tests/test-format.sh

```diff
@@ -2,18 +2,21 @@
 
 set -e
```

Comment:
the shebang sets `#!/bin/sh`. I'm not sure, but is `set -e` a bash feature?

The script output prints the following

```
Run tests/test-format.sh
  tests/test-format.sh
  shell: /usr/bin/bash -e {0}
Checking src/displayapp/DisplayApp.cpp
```

so I thought maybe the first line should be changed to `#!/bin/bash`

Reply:
I don't think that `set -e` is a bashism, or at least `shellcheck` doesn't complain about it.

Reply:
As far as I can tell, `set -e` is present in posix sh. The script passes shellcheck with no issues and works with `dash`. Ubuntu doesn't ship a posix shell by default, so it uses bash.

Reply:
then sorry about the noise